repos/spack_repo/builtin/packages/py_onnxruntime/package.py

```diff
@@ -39,12 +41,16 @@ class PyOnnxruntime(CMakePackage, PythonExtension, ROCmPackage, CudaPackage):
     depends_on("c", type="build")  # generated
     depends_on("cxx", type="build")  # generated
 
+    depends_on("binutils@2.36:", type="build")
```

Comment:
We don't normally add binutils as a dependency since most Spack packages depend on binutils in one way or another. Unless this policy has changed.